pkg/api/store/workload/transform_store.go

```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+var (
+	hideOwnerReferenceKinds = []string{"CronJob", "Deployment"}
```

Comment:
can we keep it in the map instead? This way we don't have to iterate over the slice, and do lookup better